securedrop/tests/common.py

```diff
@@ -1,120 +1,272 @@
+# -*- coding: utf-8 -*-
+
+from functools import wraps
+import gnupg
+import mock
 import os
 import shutil
-import uuid
 import subprocess
-
-import gnupg
+import time
 
 # Set environment variable so config.py uses a test environment
 os.environ['SECUREDROP_ENV'] = 'test'
+
 import config
-from db import init_db, db_session, Journalist, Reply, Source, Submission
+from db import db_session, init_db, Journalist, Reply, Source, Submission
 import crypto_util
+import store
 
-# TODO: the PID file for the redis worker is hard-coded below.
-# Ideally this constant would be provided by a test harness.
-# It has been intentionally omitted from `config.py.example`
-# in order to isolate the test vars from prod vars.
-# When refactoring the test suite, the TEST_WORKER_PIDFILE
-# TEST_WORKER_PIDFILE is also hard-coded in `manage.py`.
-TEST_WORKER_PIDFILE = "/tmp/securedrop_test_worker.pid"
 
+class TestJournalist:
+    """Test fixtures for working with :class:`db.Journalist`.
+    """
+    @staticmethod
```

Comment:
If [there's no need for `__init__()`](https://stackoverflow.com/questions/9100616/why-does-pylint-warn-about-no-init) and only static methods, why are these classes? Why is this better than [module-level functions e.g. in the relevant files](https://stackoverflow.com/questions/19620498/how-to-create-an-utility-class-correctly)? Apologies if I'm missing something here and this is a pattern I am not aware of